docs/program-setup/bare_metal/mbed2_porting.md

```diff
@@ -0,0 +1,105 @@
+# Porting an Mbed 2 target to Mbed OS 6 bare metal
+
+Mbed OS bare metal provides fewer features compared to a full RTOS platform which makes it suitable for constrained microcontrollers. This document describes how to configure an Mbed 2 target to run in bare metal mode and how to validate the port.
+
+## Configuring your target
+
+You will use the blinky bare metal Mbed OS example and make it run on your target. The configuration will be validated upon successful compilation of the example.
+
+The main steps are:
+1. Clone [mbed-os-example-blinky-baremetal](https://github.com/ARMmbed/mbed-os-example-blinky-baremetal).
+1. Edit `targets.json`. In this step, you will configure your target to run in Mbed OS 6 and then add configuration parameters for the bare metal mode. 
+1. Compile mbed-os-example-blinky-baremetal. In this step, you will validate the configuration and troubleshoot any issue.
+
+### Clone mbed-os-example-blinky-baremetal
+
+Follow these steps:
+1. git clone https://github.com/ARMmbed/mbed-os-example-blinky-baremetal
+1. cd mbed-os-example-blinky-baremetal
+
+### Edit targets.json
+
+Find your target in `targets.json` and update its configuration as described below.
+
+Configure your target to run in Mbed OS 6:
+- Remove the `release_versions` property as it is no longer necessary to specify the list of major versions of Mbed OS that the target supports.
```

Comment:
The original sentence is accurate because we no longer support Mbed 2. The suggested sentence could wrongly lead to believe that Mbed 2 is supported.

Reply:
See the updated suggestion.
The problem with the origin text is that it gives unnecessary information that seems out of context. We are talking about the changes to make and why we are making them. The suggest text explain why the change is necessary whilst the original text explains why that attribute was present.

Reply:
How about we just say `Remove the `release_versions` property as it is no longer required`?